src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -10,13 +10,10 @@ public abstract class ListCommand extends Command {
     public static final String COMMAND_WORD = "list";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": lists owners or pets in the address book whose "
-            + "relevant fields contain the specified keywords (case-insensitive) and displays them as a list with "
-            + "index numbers.\n"
+            + "and displays them as a list with index numbers.\n"
```

Comment:
This change is allowed due to the following

<img width="557" alt="image" src="https://github.com/user-attachments/assets/dbbb7bce-4b1d-4857-b2bd-a26c020842ea">